File: Lab1/Ejercicios/Ejercicio1/Ejercicio1.java
public class Ejercicio1 {

    public static int suma_array(int[] arr){
        int suma = 0;
        for(int i = 0; i < arr.length; i++){
            suma += arr[i];
        }
        return suma;
    }
    //Funcion principal.
    public static void main(String[] args) {

        int[] arr = {1, 2, 3, 4, 5};
        System.out.println("Suma - elementos: " + suma_array(arr));

    }
}

File: Lab1/Ejercicios/Ejercicio1/Ejercicio1.cpp
#include <iostream>
using namespace std;

//Uso de plantillas
template<typename T>
T suma_array(T arr[], int n) {
    T suma = 0;
    
    for(int i = 0; i < n; i++) {
        suma += arr[i];
    }
    
    return suma;
}

int main() {
    //Declaracion del arreglo.
    int arr[] = {1, 2, 3, 4, 5};
    
    cout << "Suma - Elementos: " << suma_array(arr, 5) << endl;
    return 0;
}
